### DISCUSSION TITLE
nonebot on serverless

### DISCUSSION THREAD

**Discussion by VXenomac** [nonebot/discussions#321] [问答]
我想将 nonebot 以函数计算的模式进行部署，应该如何做呢？

**Comment by yanyongyu** [comment]
你可以查找一下asgi的serverless部署方式（如阿里云serverless），不过目前nb暂不支持具有会话状态的serverless（即无法保存状态），后续可能可以靠redis之类的数据库实现。

可以通过如下代码获取asgi对象

```python
import nonebot

asgi = nonebot.get_asgi()
```
